requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,51 @@
+## User Roles
+
+### Unauthorized user
+
+Unauthorized users can sign in, view list of internships, but cannot save their progress.
+
+### Authorized user
+
+Authorized users can view list of internships sorted by interest, save their progress, follow other users and company, sign out.
+
+### Company member
+
+Company member can create new internship announcements.
+
+### Moderator
+
+Moderator can view, edit and delete internship announcements.
+
+## User Case Diagram
+
+<p align="center">
+  <img src="/images/use_case.png" title="Use Case" alt="Use Case">
+</p>
+
+## User Story
+
+#### User story №1
+
+As an unauthorized user, I want to see the list of available internships and companies to receive information about them.
+
+#### User story №2
+
+As an authorized user, I want to choose suitable internships in order to apply to them.
+
+#### User story №3
+
+As a member of a company, I want to add internships to the list so that users will be able to apply to them.
+
+## Job Story
+
+#### Job Story №1
+
+When I search for internships, I want to first see opportunities that meet all of my criteria so that I do not have to waste my time on scrolling.
+
+#### Job Story №2
+
+When I research one particular internship, I want to see all the important information such as dates, duration e.t.c. so that I do not have to search for additional information.
+
+#### Job Story №3
+
+When I want to organize an internship, I want to create the internship opportunity in the app in order to find candidates.
```

Comment:
Job story is about the wish and the result. In that case, you are talking about the way your application works -- it was created to find candidates and for it you have to place internship opportunity. Instead, you should talk about the specific thing that the client wants to achieve. For example, make possible candidates know about the internship.